okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -842,6 +842,19 @@ public String fragment() {
   }
 
   /**
+   * Returns the redacted URL that would only show the domain name.
```

Comment:
Not sure about this comment. It suggests the `/...` won’t be present . . . maybe say `Returns a URL like http://example.com/... with the username, password, path, query, and fragment stripped` ?